layouts/dashboard.tsx

```diff
@@ -1,19 +1,5 @@
-import { Box, Flex } from "@artsy/palette";
-import { BareHeader } from "../components/BareHeader";
-import { Sidebar } from "../components/dashboard/Sidebar";
+import { Layout } from "../pages/dashboard/_layout";
```

Comment:
#minor but are semis necessary? We don't used them anywhere else all across artsy.

Reply:
Oh, I just haven't changed the default prettier config. Will do that in a separate PR to reduce noise in this one.